contrib/hbwin/hbwin.hbp

```diff
@@ -29,7 +29,7 @@ mapi.c
 oemansi.c
 olecore.c
 oleuuid.c{watcom}
-hbolesrv.c
+{HBWIN_HAS_OLESRV}hbolesrv.c
```

Comment:
No Viktor I wasn't asking you to help me with something local. I just built a lib called hbwin32  with additional apis I needed at the time.
Instead of starting from scratch I used the hbwin contrib as a template for it.

My hbwin32.hbc has the line

libs=${_HB_DYNPREF}${hb_name}${_HB_DYNSUFF}

I copied from hbwin.hbc, but never used _HB_DYN* env var